Python/Bollywood.py

```diff
@@ -0,0 +1,47 @@
+import random
+movies = ('LAGAAN', 'HIGHWAY', 'THOR', 'DANGAL', 'NEWTON','ANEK','APNE','CHHICHHORE','WAR','GULAAL','DHOOM','MALANG', 'STAR WARS') #movies_name_that_you_will_get_to_play
+selected_movie = random.sample(movies,1)    #one movie is selected by random library
+k = '*' * len(selected_movie[0])
```

Comment:
Please change this variable (k) to better name

Reply:
okay